borsh-derive/src/internals/deserialize/enum/mod.rs

```diff
@@ -161,7 +161,7 @@ pub fn enum_de(input: &ItemEnum, cratename: Ident) -> syn::Result<TokenStream2>
 
 #[cfg(test)]
 mod tests {
-    use crate::test_helpers::pretty_print_syn_str;
+    use crate::internals::test_helpers::pretty_print_syn_str;
```

Comment:
Add import 
```rust
use crate::internals::test_helpers::{local_insta_assert_debug_snapshot, local_insta_assert_snapshot};
```
Replace `insta::assert_snapshot!` => `local_insta_assert_snapshot!` ,
`insta::assert_debug_snapshot!` => `local_insta_assert_debug_snapshot!` everywhere in this `tests` module to shorten snapshot names.
(see [commit](https://github.com/near/borsh-rs/pull/185/commits/2fe3cfef30991fbd8afc4106d33e66fb15c6176a) )